### ISSUE TITLE
Question on normalisation in faster_rcnn

### ISSUE THREAD

**Issue by robmarkcole** [allenai/rslearn#171]
I'm trying to resolve differences between the implementation here and the one in TorchGeo. One of the main differences is in the normalisation - [here](https://github.com/allenai/rslearn/blob/80266412a2a9b5dcf6c03e516019f173d9413fdd/rslearn/models/faster_rcnn.py#L10C1-L24C10) you always resize to 800, even though the patch_size used is 512. 
Why is that?

Thanks

```python
class NoopTransform(torch.nn.Module):
    """A placeholder transform used with torchvision detection model."""

    def __init__(self) -> None:
        """Create a new NoopTransform."""
        super().__init__()

        self.transform = (
            torchvision.models.detection.transform.GeneralizedRCNNTransform(
                min_size=800,
                max_size=800,
                image_mean=[],
                image_std=[],
            )
        )
```

**Comment by favyen2** [comment]
The min_size/max_size is ignored since we call self.transform.batch_images(...) directly rather than self.transform(...). See here:

https://github.com/pytorch/vision/blob/5f03dc524bdb7529bb4f2e84d2d8c237233fc62a/torchvision/models/detection/transform.py#L237

I will add a comment here.

**Comment by favyen2** [comment]
closing since it will be tracked in #183
